include/swift/Reflection/ReflectionContext.h

```diff
@@ -378,30 +378,37 @@ class ReflectionContext
     return readPECOFFSections(ImageStart);
   }
 
-  template <typename T> bool readELFSections(RemoteAddress ImageStart) {
+  template <typename T>
+  bool readELFSections(RemoteAddress ImageStart, const void *FileBuffer,
+                       uint64_t FileSize) {
```

Comment:
Can you add a Doxygen comment explaining what FileBuffer is supposed to contain and why it is needed?

Reply:
Ok, added it on top of `readELF`, as that's the more general function that should be called.